internal/service/lightsail/database.go

```diff
@@ -0,0 +1,462 @@
+)
+
+func ResourceDatabase() *schema.Resource {
+	return &schema.Resource{
+		Create: ResourceDatabaseCreate,
+		Read:   ResourceDatabaseRead,
+		Update: ResourceDatabaseUpdate,
+		Delete: ResourceDatabaseDelete,
+		Importer: &schema.ResourceImporter{
+			State: ResourceDatabaseImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"master_database_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"master_password": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+			"master_username": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"blueprint_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"bundle_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			// Optional attributes
+			"preferred_backup_window": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"preferred_maintenance_window": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"publicly_accessible": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"apply_immediately": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"backup_retention_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"skip_final_snapshot": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"final_snapshot_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			// additional info returned from the API
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ca_certificate_identifier": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"engine": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"engine_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"cpu_count": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"ram_size": {
+				Type:     schema.TypeFloat,
+				Computed: true,
+			},
+			"disk_size": {
+				Type:     schema.TypeFloat,
+				Computed: true,
+			},
+			"master_endpoint_port": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"master_endpoint_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"secondary_availability_zone": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"support_code": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tags": tftags.TagsSchema(),
```

Comment:
Should have Tags and TagsAll. See reference here:
https://github.com/hashicorp/terraform-provider-aws/blob/a7f0bd58b9752f8dd7d738239515329c5042c28e/internal/service/lightsail/instance.go#L123

Reply:
Added the `tags_all` field.

Reply:
added the field to the documentation as well